server/keyshare/tasks/tasks.go

```diff
@@ -83,6 +83,11 @@ func (t *taskHandler) sendExpiryEmails(id int64, username, lang string) error {
 		func(emailRes *sql.Rows) error {
 			var email string
 			err := emailRes.Scan(&email)
+			// FIXME: 'return nil' will prevent abortion of 'expireAccounts()'
+			// but will not take care of the actual problem of handling the invalid email
+			if err == keyshare.ErrInvalidEmail {
+				return nil
+			}
```

Comment:
Now I checked it better, you added the error handling at the wrong place. `emailRes.Scan(&email)` returns errors from the database. That will never give `keyshare.ErrInvalidEmail`. You should have checked this after `t.conf.SendEmail`.